api/v1/config.js

```diff
@@ -1,11 +1,32 @@
 module.exports = {
   posts: [
     {
-      date: '2020-06-14T00:00:00.000Z',
+      date: '2020-06-20T09:35:00.000Z',
```

Comment:
Probably best to not change dates once they have been in prod for a while even if they were wrong. For people that index things mentally.

Reply:
When publishing the website, I did not know what day it would actually go live, this reflects the actual date.